src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -60,6 +66,10 @@ The release notes include:
 
    -  {:.fix}Renamed the `./bin/docker` file to `./bin/magento-docker` to fix an issue that caused some Docker environments to break because the `./bin/docker` file overwrites existing Docker binary files. This is a [backward incompatible change] that requires updates to your scripts and commands.<!-- MAGECLOUD-4038 -->
 
+   -  {:.new}**Added an option to expose the database port to the host**–[Updated submitted by Adarsh Manickam](https://github.com/magento/magento-cloud-docker/pull/101).
+
+      Use the `--expose-db-port=<PORT>` option to expose the database port to the host when building the `docker-compose.yml` file: `bin/ece-docker build:compose --expose-db-port=<PORT>`<!--MAGECLOUD-4454-->
```

Comment:
Formatting here seems different than the rest of the items in the list. Seems like this line belongs before the "Update submitted by..."

Reply:
was going back and forth on where to put the contributor acknowledgement-- I think I'll move them all to the end of the release note.